examples/meta/generator/targets/cpp.json

```diff
@@ -59,8 +59,8 @@
         "RealFeatures": "DenseFeatures<float64_t>",
         "RealSubsetFeatures": "DenseSubsetFeatures<float64_t>",
         "StringCharFeatures": "StringFeatures<char>",
-        "StringCharList": "SGStringList<char>",
-        "StringWordList": "SGStringList<uint16_t>",
+        "StringCharList": "std::vector<SGVector<char>>",
```

Comment:
i wonder whether we should rename "List" to "Vector"?
But this is minor

Reply:
I know this just for the meta language, but "List" is more language neutral and common than "Vector". The equivalent of vector in java is ArrayList, and in C# is List.

Reply:
good points and I agree! Let's keep it!